debugAttach fails on Android with new clang versions

On Android, `util.h` defines `boolean debugAttach;`, and that header is added to all services.
When linking those services, this exception is thrown:
```
[INFO] ==================== LINK TASK ====================
[INFO] [SUB] ~/.gluon/substrate/Android/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/../lib/gcc/aarch64-linux-android/4.9.x/../../../../aarch64-linux-android/bin/ld.gold: error: /Sample/target/client/aarch64-android/gvm/lib/libutil.a(util.o): multiple definition of 'debugAttach'
[INFO] [SUB] /~/.gluon/substrate/Android/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/../lib/gcc/aarch64-linux-android/4.9.x/../../../../aarch64-linux-android/bin/ld.gold: /Sample/target/client/aarch64-android/gvm/lib/libaccelerometer.a(accelerometer.o): previous definition here
```

This happens with clang 11.0.5, but not with older versions (9.0.8)

We need to move the constant `debugAttach` to a function `isDebugAttach()`.

--- a/modules/audio/src/main/native/android/c/audio.c
+++ b/modules/audio/src/main/native/android/c/audio.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020 Gluon
+ * Copyright (c) 2020, 2021, Gluon
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -86,7 +86,7 @@ JNIEXPORT jint JNICALL Java_com_gluonhq_attach_audio_impl_AndroidAudioService_lo
 {
     const char *jURLChars = (*env)->GetStringUTFChars(env, jURL, NULL);
 
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Loading sound from file. Absolute path: %s", jURLChars);
     }
 
@@ -103,7 +103,7 @@ JNIEXPORT jint JNICALL Java_com_gluonhq_attach_audio_impl_AndroidAudioService_lo
 {
     const char *jURLChars = (*env)->GetStringUTFChars(env, jURL, NULL);
 
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Loading music from file. Absolute path: %s", jURLChars);
     }
 

--- a/modules/augmented-reality/src/main/native/android/c/augmentedreality.c
+++ b/modules/augmented-reality/src/main/native/android/c/augmentedreality.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, Gluon
+ * Copyright (c) 2020, 2021, Gluon
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -86,20 +86,20 @@ JNI_OnLoad_augmentedreality(JavaVM *vm, void *reserved)
 // from Java to Android
 JNIEXPORT jstring JNICALL Java_com_gluonhq_attach_augmentedreality_impl_AndroidAugmentedRealityService_checkAR
 (JNIEnv *env, jclass jClass) {
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("AugmentedReality check AR\n");
     }
     ATTACH_DALVIK();
     jstring answer = (*dalvikEnv)->CallObjectMethod(dalvikEnv, jDalvikAugmentedRealityService, jAugmentedRealityServiceCheckARMethod);
     if (answer == NULL) {
-        if (debugAttach) {
+        if (isDebugAttach()) {
             ATTACH_LOG_FINE("Error: AugmentedReality:: availability is null\n");
         }
         DETACH_DALVIK();
         return NULL;
     }
     const char *answerChars = (*dalvikEnv)->GetStringUTFChars(dalvikEnv, answer, 0);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("AugmentedReality availability is %s", answerChars);
     }
     DETACH_DALVIK();
@@ -108,7 +108,7 @@ JNIEXPORT jstring JNICALL Java_com_gluonhq_attach_augmentedreality_impl_AndroidA
 
 JNIEXPORT void JNICALL Java_com_gluonhq_attach_augmentedreality_impl_AndroidAugmentedRealityService_showNativeAR
 (JNIEnv *env, jclass jClass) {
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("AugmentedReality show AR");
     }
     ATTACH_DALVIK();
@@ -118,7 +118,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_attach_augmentedreality_impl_AndroidAugm
 
 JNIEXPORT void JNICALL Java_com_gluonhq_attach_augmentedreality_impl_AndroidAugmentedRealityService_enableDebugAR
 (JNIEnv *env, jclass jClass, jboolean enable) {
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("AugmentedReality enable debug AR");
     }
     ATTACH_DALVIK();
@@ -128,7 +128,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_attach_augmentedreality_impl_AndroidAugm
 
 JNIEXPORT void JNICALL Java_com_gluonhq_attach_augmentedreality_impl_AndroidAugmentedRealityService_setARModel
 (JNIEnv *env, jclass jClass, jstring jobjfile, jstring jtexturefile, jdouble scale) {
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("AugmentedReality AR model");
     }
     const char *objfileChars = (*env)->GetStringUTFChars(env, jobjfile, NULL);
@@ -151,7 +151,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_attach_augmentedreality_impl_AndroidAugm
 JNIEXPORT void JNICALL Java_com_gluonhq_helloandroid_DalvikAugmentedRealityService_nativeAugmentedRealityAvailability(
     JNIEnv *env, jobject service, jstring result) {
     const char *resultChars = (*env)->GetStringUTFChars(env, result, NULL);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("AugmentedReality result %s", resultChars);
     }
     ATTACH_GRAAL();
@@ -163,7 +163,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_helloandroid_DalvikAugmentedRealityServi
 
 JNIEXPORT void JNICALL Java_com_gluonhq_helloandroid_ar_ARRenderer_nativeCancelAR(
     JNIEnv *env, jobject service) {
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("AugmentedReality cancel AR");
     }
     ATTACH_GRAAL();

--- a/modules/barcode-scan/src/main/native/android/c/barcodescan.c
+++ b/modules/barcode-scan/src/main/native/android/c/barcodescan.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, Gluon
+ * Copyright (c) 2020, 2021, Gluon
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -81,7 +81,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_attach_barcodescan_impl_AndroidBarcodeSc
     const char *titleChars = (*env)->GetStringUTFChars(env, jtitle, NULL);
     const char *legendChars = (*env)->GetStringUTFChars(env, jlegend, NULL);
     const char *resulttextChars = (*env)->GetStringUTFChars(env, jresulttext, NULL);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("BarcodeScan start for title %s, legend %s, resulttext %s\n", titleChars, legendChars, resulttextChars);
     }
     ATTACH_DALVIK();
@@ -103,7 +103,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_attach_barcodescan_impl_AndroidBarcodeSc
 JNIEXPORT void JNICALL Java_com_gluonhq_helloandroid_DalvikBarcodeScanService_nativeBarcodeScanResult(
     JNIEnv *env, jobject service, jstring result) {
     const char *resultChars = (*env)->GetStringUTFChars(env, result, NULL);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("BarcodeScan result %s\n", resultChars);
     }
     ATTACH_GRAAL();

--- a/modules/battery/src/main/native/android/c/battery.c
+++ b/modules/battery/src/main/native/android/c/battery.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020 Gluon
+ * Copyright (c) 2020, 2021, Gluon
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -76,7 +76,7 @@ JNI_OnLoad_battery(JavaVM *vm, void *reserved)
 
 JNIEXPORT void JNICALL Java_com_gluonhq_helloandroid_DalvikBatteryService_nativeBattery(
     JNIEnv *env, jobject service, jfloat flevel, jboolean bstatus) {
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Native layer got new battery level: %.2f status %d\n", flevel, (bstatus ? 1 : 0));
     }
     ATTACH_GRAAL();

--- a/modules/ble/src/main/native/android/c/ble.c
+++ b/modules/ble/src/main/native/android/c/ble.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020 Gluon
+ * Copyright (c) 2020, 2021, Gluon
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -270,7 +270,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_attach_ble_impl_AndroidBleService_doSubs
 // BLE BEACONS
 
 JNIEXPORT void JNICALL Java_com_gluonhq_helloandroid_DalvikBleService_scanDetected(JNIEnv *env, jobject service, jstring uuid, jint major, jint minor, jint ris, jint proxy) {
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Scan Detection is now in native layer, major = %d\n", major);
     }
     const char *uuidChars = (*env)->GetStringUTFChars(env, uuid, NULL);
@@ -288,7 +288,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_helloandroid_DalvikBleService_scanDevice
         jstring name, jstring address) {
     const char *nameChars = (*env)->GetStringUTFChars(env, name, NULL);
     const char *addressChars = (*env)->GetStringUTFChars(env, address, NULL);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Scan Device Detection, name = %s, address = %s\n", nameChars, addressChars);
     }
 
@@ -306,7 +306,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_helloandroid_BleGattCallback_setState(JN
         jstring name, jstring state) {
     const char *nameChars = (*env)->GetStringUTFChars(env, name, NULL);
     const char *stateChars = (*env)->GetStringUTFChars(env, state, NULL);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Device state, name = %s, state = %s\n", nameChars, stateChars);
     }
     ATTACH_GRAAL();
@@ -323,7 +323,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_helloandroid_BleGattCallback_addProfile(
     const char *nameChars = (*env)->GetStringUTFChars(env, name, NULL);
     const char *uuidChars = (*env)->GetStringUTFChars(env, uuid, NULL);
     const char *typeChars = (*env)->GetStringUTFChars(env, type, NULL);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Device type, name = %s, service: uuid = %s, type = %s\n", nameChars, uuidChars, typeChars);
     }
     ATTACH_GRAAL();
@@ -343,7 +343,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_helloandroid_BleGattCallback_addCharacte
     const char *profileUuidChars = (*env)->GetStringUTFChars(env, profileUuid, NULL);
     const char *charUuidChars = (*env)->GetStringUTFChars(env, charUuid, NULL);
     const char *propertiesChars = (*env)->GetStringUTFChars(env, properties, NULL);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Device name = %s, service: profileUuid = %s, charUuid = %s, properties = %s\n",
             nameChars, profileUuidChars, charUuidChars, propertiesChars);
     }
@@ -369,7 +369,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_helloandroid_BleGattCallback_addDescript
     const char *descUuidChars = (*env)->GetStringUTFChars(env, descUuid, NULL);
     jbyte *valueBytes = (*env)->GetByteArrayElements(env, value, NULL);
     int size = (*env)->GetArrayLength(env, value);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Device name = %s, service: profileUuid = %s, charUuid = %s, descUuid = %s\n",
             nameChars, profileUuidChars, charUuidChars, descUuidChars);
     }

--- a/modules/connectivity/src/main/native/android/c/connectivity.c
+++ b/modules/connectivity/src/main/native/android/c/connectivity.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020 Gluon
+ * Copyright (c) 2020, 2021, Gluon
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -76,7 +76,7 @@ JNI_OnLoad_connectivity(JavaVM *vm, void *reserved)
 
 JNIEXPORT void JNICALL Java_com_gluonhq_helloandroid_DalvikConnectivityService_nativeConnectivity(
     JNIEnv *env, jobject service, jboolean jconnected) {
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Native layer got new connectivity: %d\n", (jconnected ? 1 : 0));
     }
     ATTACH_GRAAL();

--- a/modules/device/src/main/native/android/c/device.c
+++ b/modules/device/src/main/native/android/c/device.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020 Gluon
+ * Copyright (c) 2020, 2021, Gluon
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -87,7 +87,7 @@ JNI_OnLoad_device(JavaVM *vm, void *reserved)
 JNIEXPORT jobject JNICALL Java_com_gluonhq_attach_device_impl_AndroidDeviceService_getDeviceInfo
 (JNIEnv *env, jclass jClass)
 {
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Retrieving DeviceInfo\n");
     }
 
@@ -103,7 +103,7 @@ JNIEXPORT jobject JNICALL Java_com_gluonhq_attach_device_impl_AndroidDeviceServi
     jboolean wearable = (*dalvikEnv)->CallBooleanMethod(dalvikEnv, jDalvikDeviceService, jDeviceServiceIsWearable);
     DETACH_DALVIK();
 
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Retrieved DeviceInfo: model=%s, uuid=%s, platform=%s, version=%s, wearable=%d\n",
                 responseModelChars, responseUuidChars, responsePlatformChars, responseVersionChars, wearable);
     }

--- a/modules/local-notifications/src/main/native/android/c/localnotifications.c
+++ b/modules/local-notifications/src/main/native/android/c/localnotifications.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, Gluon
+ * Copyright (c) 2020, 2021, Gluon
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -125,7 +125,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_helloandroid_NotificationActivity_proces
     (JNIEnv *env, jobject service, jstring jkey, jstring jvalue) {
     const char *keyChars = (*env)->GetStringUTFChars(env, jkey, NULL);
     const char *valueChars = (*env)->GetStringUTFChars(env, jvalue, NULL);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("LocalNotifications:: Native layer got key: %s, value: %s", keyChars, valueChars);
     }
     ATTACH_GRAAL();

--- a/modules/orientation/src/main/native/android/c/orientation.c
+++ b/modules/orientation/src/main/native/android/c/orientation.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020 Gluon
+ * Copyright (c) 2020, 2021, Gluon
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -77,7 +77,7 @@ JNI_OnLoad_orientation(JavaVM *vm, void *reserved)
 JNIEXPORT void JNICALL Java_com_gluonhq_helloandroid_DalvikOrientationService_nativeOrientation(
     JNIEnv *env, jobject service, jstring jorientation) {
     const char *chars = (*env)->GetStringUTFChars(env, jorientation, NULL);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Native layer got new orientation: %s\n", chars);
     }
     ATTACH_GRAAL();

--- a/modules/pictures/src/main/native/android/c/pictures.c
+++ b/modules/pictures/src/main/native/android/c/pictures.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, Gluon
+ * Copyright (c) 2020, 2021, Gluon
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -101,7 +101,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_attach_pictures_impl_AndroidPicturesServ
 ///////////////////////////
 
 JNIEXPORT void JNICALL Java_com_gluonhq_helloandroid_DalvikPicturesService_sendPhotoFile(JNIEnv *env, jobject service, jstring path, jint rotate) {
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Send Photo File\n");
     }
     const char *pathChars = (*env)->GetStringUTFChars(env, path, NULL);

--- a/modules/push-notifications/src/main/native/android/c/pushnotifications.c
+++ b/modules/push-notifications/src/main/native/android/c/pushnotifications.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, Gluon
+ * Copyright (c) 2020, 2021, Gluon
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -117,7 +117,7 @@ JNIEXPORT jstring JNICALL Java_com_gluonhq_attach_pushnotifications_impl_Android
     jstring dalvikErrorString = (jstring) (*dalvikEnv)->CallObjectMethod(dalvikEnv, jDalvikPushNotificationsService, jDalvikPushNotificationsServiceGetErrorString,
                 resultCode);
     const char *errorStringChars = (*dalvikEnv)->GetStringUTFChars(dalvikEnv, dalvikErrorString, NULL);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("PushNotification error string: %s", errorStringChars);
     }
     jstring graalErrorString = (*env)->NewStringUTF(env, errorStringChars);
@@ -132,7 +132,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_attach_pushnotifications_impl_AndroidPus
     const char *projectNumberChars = (*env)->GetStringUTFChars(env, projectNumber, NULL);
     const char *projectIdChars = (*env)->GetStringUTFChars(env, projectId, NULL);
     const char *apiKeyChars = (*env)->GetStringUTFChars(env, apiKey, NULL);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("PushNotification::initializeFirebase with app Id: %s, project number: %s, project id: %s, and api key: %s", applicationIdChars, projectNumberChars, projectIdChars, apiKeyChars);
     }
     ATTACH_DALVIK();
@@ -160,7 +160,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_attach_pushnotifications_impl_AndroidPus
 JNIEXPORT void JNICALL Java_com_gluonhq_helloandroid_PushFcmMessagingService_sendToken
     (JNIEnv *env, jobject service, jstring jtoken) {
     const char *tokenChars = (*env)->GetStringUTFChars(env, jtoken, NULL);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("PushNotifications:: Native layer got token: %s", tokenChars);
     }
     ATTACH_GRAAL();
@@ -175,7 +175,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_helloandroid_DalvikPushNotificationsServ
     (JNIEnv *env, jobject service, jstring jkey, jstring jvalue) {
     const char *keyChars = (*env)->GetStringUTFChars(env, jkey, NULL);
     const char *valueChars = (*env)->GetStringUTFChars(env, jvalue, NULL);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("PushNotifications:: Native layer got key: %s, value: %s", keyChars, valueChars);
     }
     ATTACH_GRAAL();

--- a/modules/runtime-args/src/main/native/android/c/runtimeargs.c
+++ b/modules/runtime-args/src/main/native/android/c/runtimeargs.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020 Gluon
+ * Copyright (c) 2020, 2021, Gluon
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -85,7 +85,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_helloandroid_LaunchURLActivity_processRu
     (JNIEnv *env, jobject service, jstring jkey, jstring jvalue) {
     const char *keyChars = (*env)->GetStringUTFChars(env, jkey, NULL);
     const char *valueChars = (*env)->GetStringUTFChars(env, jvalue, NULL);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("RuntimeArgs:: Native layer got key: %s, value: %s", keyChars, valueChars);
     }
     ATTACH_GRAAL();

--- a/modules/settings/src/main/native/android/c/settings.c
+++ b/modules/settings/src/main/native/android/c/settings.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020 Gluon
+ * Copyright (c) 2020, 2021, Gluon
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -79,7 +79,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_attach_settings_impl_AndroidSettingsServ
     ATTACH_DALVIK();
     jstring dkey = (*dalvikEnv)->NewStringUTF(dalvikEnv, keyChars);
     jstring dvalue = (*dalvikEnv)->NewStringUTF(dalvikEnv, valueChars);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Storing settings for = %s, %s\n", keyChars, valueChars);
     }
     (*dalvikEnv)->CallVoidMethod(dalvikEnv, jDalvikSettingsService, jSettingsServiceStore, dkey, dvalue);
@@ -94,7 +94,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_attach_settings_impl_AndroidSettingsServ
     const char *keyChars = (*env)->GetStringUTFChars(env, jkey, NULL);
     ATTACH_DALVIK();
     jstring dkey = (*dalvikEnv)->NewStringUTF(dalvikEnv, keyChars);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Remove settings for = %s\n", keyChars);
     }
     (*dalvikEnv)->CallVoidMethod(dalvikEnv, jDalvikSettingsService, jSettingsServiceRemove, dkey);
@@ -108,19 +108,19 @@ JNIEXPORT jstring JNICALL Java_com_gluonhq_attach_settings_impl_AndroidSettingsS
     const char *keyChars = (*env)->GetStringUTFChars(env, jkey, NULL);
     ATTACH_DALVIK();
     jstring dkey = (*dalvikEnv)->NewStringUTF(dalvikEnv, keyChars);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Retrieving settings for = %s\n", keyChars);
     }
     jstring answer = (*dalvikEnv)->CallObjectMethod(dalvikEnv, jDalvikSettingsService, jSettingsServiceRetrieve, dkey);
     if (answer == NULL) {
-        if (debugAttach) {
+        if (isDebugAttach()) {
             ATTACH_LOG_FINE("Error: Settings for = %s not found\n", keyChars);
         }
         DETACH_DALVIK();
         return NULL;
     }
     const char *answerChars = (*dalvikEnv)->GetStringUTFChars(dalvikEnv, answer, 0);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Retrieved settings for = %s\n", answerChars);
     }
     DETACH_DALVIK();

--- a/modules/share/src/main/native/android/c/share.c
+++ b/modules/share/src/main/native/android/c/share.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020 Gluon
+ * Copyright (c) 2020, 2021, Gluon
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -77,7 +77,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_attach_share_impl_AndroidShareService_sh
     ATTACH_DALVIK();
     jstring dsubject = (*dalvikEnv)->NewStringUTF(dalvikEnv, subjectChars);
     jstring dcontent = (*dalvikEnv)->NewStringUTF(dalvikEnv, contentChars);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Share text, subject = %s, content = %s\n", subjectChars, contentChars);
     }
     (*dalvikEnv)->CallVoidMethod(dalvikEnv, jDalvikShareService, jShareServiceShareText, dsubject, dcontent);
@@ -98,7 +98,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_attach_share_impl_AndroidShareService_sh
     jstring dcontent = (*dalvikEnv)->NewStringUTF(dalvikEnv, contentChars);
     jstring dtype = (*dalvikEnv)->NewStringUTF(dalvikEnv, typeChars);
     jstring dfilename = (*dalvikEnv)->NewStringUTF(dalvikEnv, filenameChars);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Share file, subject = %s, content = %s, type = %s, filename = %s\n",
             subjectChars, contentChars, typeChars, filenameChars);
     }

--- a/modules/statusbar/src/main/native/android/c/statusbar.c
+++ b/modules/statusbar/src/main/native/android/c/statusbar.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020 Gluon
+ * Copyright (c) 2020, 2021, Gluon
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -71,7 +71,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_attach_statusbar_impl_AndroidStatusBarSe
 (JNIEnv *env, jclass jClass, jint color)
 {
     ATTACH_DALVIK();
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Set native color, value: %d", color);
     }
     (*dalvikEnv)->CallVoidMethod(dalvikEnv, jDalvikStatusBarService, jStatusBarServiceColorMethod, color);

--- a/modules/storage/src/main/native/android/c/storage.c
+++ b/modules/storage/src/main/native/android/c/storage.c
@@ -77,12 +77,12 @@ JNIEXPORT jstring JNICALL Java_com_gluonhq_attach_storage_impl_AndroidStorageSer
     const char *directoryChars = (*env)->GetStringUTFChars(env, jdirectory, NULL);
     ATTACH_DALVIK();
     jstring ddirectory = (*dalvikEnv)->NewStringUTF(dalvikEnv, directoryChars);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Retrieving external storage for = %s\n", directoryChars);
     }
     jstring answer = (*dalvikEnv)->CallObjectMethod(dalvikEnv, jDalvikStorageService, jStorageServicePublicStorage, ddirectory);
     const char *answerChars = (*dalvikEnv)->GetStringUTFChars(dalvikEnv, answer, 0);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Retrieved external storage at = %s\n", answerChars);
     }
     DETACH_DALVIK();

--- a/modules/util/src/main/native/android/c/util.c
+++ b/modules/util/src/main/native/android/c/util.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, Gluon
+ * Copyright (c) 2020, 2021, Gluon
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -38,6 +38,7 @@ static jmethodID jUtilOnLifecycleEventMethod;
 static jmethodID jUtilRequestPermissionsMethod;
 
 static jboolean initialized;
+static jboolean debugAttach;
 
 // Attach classes
 static jclass jAccelerometerServiceClass;
@@ -244,6 +245,10 @@ JNI_OnLoad_util(JavaVM *vm, void *reserved)
 
 JavaVM* getGraalVM() {
     return graalVM;
+}
+
+jboolean isDebugAttach() {
+    return debugAttach;
 }
 
 // from Java to Android

--- a/modules/util/src/main/native/android/c/util.h
+++ b/modules/util/src/main/native/android/c/util.h
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, Gluon
+ * Copyright (c) 2020, 2021, Gluon
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -41,4 +41,4 @@ JavaVM* getGraalVM();
 
 jclass substrateGetUtilClass();
 
-jboolean debugAttach;
+jboolean isDebugAttach();

--- a/modules/util/src/main/native/android/c/util.h
+++ b/modules/util/src/main/native/android/c/util.h
@@ -46,4 +46,4 @@ jclass substrateGetUtilClass();
 
 jboolean isDebugAttach();
 
-#endif
+#endif // ATTACH_UTIL

--- a/modules/util/src/main/native/android/c/util.h
+++ b/modules/util/src/main/native/android/c/util.h
@@ -26,6 +26,9 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#ifndef ATTACH_UTIL
+#define ATTACH_UTIL
+
 #include <android/native_window_jni.h>
 #include "attach_macros_android.h"
 
@@ -42,3 +45,5 @@ JavaVM* getGraalVM();
 jclass substrateGetUtilClass();
 
 jboolean isDebugAttach();
+
+#endif

--- a/modules/vibration/src/main/native/android/c/vibration.c
+++ b/modules/vibration/src/main/native/android/c/vibration.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, Gluon
+ * Copyright (c) 2020, 2021, Gluon
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -68,7 +68,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_attach_vibration_impl_AndroidVibrationSe
     (JNIEnv *env, jclass jClass, jlongArray jpattern)
 {
     int count = (*env)->GetArrayLength(env, jpattern);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Vibrate pattern with %d items", count);
     }
 

--- a/modules/video/src/main/native/android/c/video.c
+++ b/modules/video/src/main/native/android/c/video.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020 Gluon
+ * Copyright (c) 2020, 2021, Gluon
  *
  * This program is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -104,7 +104,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_attach_video_impl_AndroidVideoService_se
 (JNIEnv *env, jclass jClass, jobjectArray jPlaylistArray)
 {
     int playlistCount = (*env)->GetArrayLength(env, jPlaylistArray);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Adding video playlist with %d items", playlistCount);
     }
     ATTACH_DALVIK();
@@ -203,7 +203,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_attach_video_impl_AndroidVideoService_se
 
     const char *alignmentHChars = (*env)->GetStringUTFChars(env, jalignmentH, NULL);
     const char *alignmentVChars = (*env)->GetStringUTFChars(env, jalignmentV, NULL);
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Video Alignment H: %s, V: %s", alignmentHChars, alignmentVChars);
     }
     ATTACH_DALVIK();
@@ -222,7 +222,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_attach_video_impl_AndroidVideoService_se
 
 JNIEXPORT void JNICALL Java_com_gluonhq_helloandroid_DalvikVideoService_nativeStatus
     (JNIEnv *env, jobject service, jint status) {
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Media Player Status: %d", status);
     }
     ATTACH_GRAAL();
@@ -239,7 +239,7 @@ JNIEXPORT void JNICALL Java_com_gluonhq_helloandroid_DalvikVideoService_nativeFu
 
 JNIEXPORT void JNICALL Java_com_gluonhq_helloandroid_DalvikVideoService_nativeCurrentIndex
     (JNIEnv *env, jobject service, jint index) {
-    if (debugAttach) {
+    if (isDebugAttach()) {
         ATTACH_LOG_FINE("Media Player index: %d", index);
     }
     ATTACH_GRAAL();

